configs/eslint-config-compass/index.js

```diff
@@ -44,8 +44,7 @@ const tsxRules = {
   'react-hooks/exhaustive-deps': [
     'warn',
     {
-      additionalHooks:
-        '(useTrackOnChange|useContextMenuItems|useContextMenuGroups)',
+      additionalHooks: '(useTrackOnChange|useContextMenuGroups)',
```

Comment:
we seem to be converging on `useContextMenuGroups`  so I'm removing the old `useContextMenuItems` syntax